share/spack/gitlab/cloud_pipelines/stacks/aws-pcluster-neoverse_n1/spack.yaml

```diff
@@ -0,0 +1,61 @@
+spack:
+  view: false
+
+  definitions:
+  - compiler_specs:
```

Comment:
Since this stuff is identical between the stacks, but may be worth while to merge the aarch64 stacks and the x86_64_v4 and use a matrix spec and submappings to assign the runner tags. appropriately.

```yaml
ci:
  pipeline-gen:
  - match_behavior: first
    submapping
     - match:
       - target=neoverse_n1
         build_job:
           tags: ["graviton2"]
     - match:
       - target=neoverse_v1
         build_job:
           tags: ["graviton3"]
```

I am not sure if both `neoverse_n1` and `neoverse_v1` can be concretized on gravitron3 runners, but I am pretty sure icelake and skylake can be so I would think the same backwards compatibility might apply to ARM architectures.

Reply:
Let me take a look at this after the deadline. It seems I need to learn some more syntax.